docs/using-lagoon-the-basics/lagoon-build-errors-and-warnings.md

```diff
@@ -1,6 +1,8 @@
 # Lagoon Build Errors and Warnings
 
-Newer releases of Lagoon have the capability to identify build errors, and highlight them as warnings without failing the build. They should be resolved wherever possible, as future releases of Lagoon may not be able to handle the errors.
+Newer releases of Lagoon have the capability to identify potential issues with the build, and highlight them as warnings without failing. This is also a way for the Lagoon team to inform users of pending deprecations and changes to functionality.
+
+For example, if the Lagoon team makes a change to a setting in `lagoon.yml`, and there’s something that users need to change, the warning will mention it, so users will be able to change it before it becomes a breaking change. They should be resolved ASAP wherever possible, as future releases of Lagoon may not be able to handle the errors, but they shouldn'y stop your build.
```

Comment:
Typo with `shouldn't`

Reply:
Fixed, thanks! It's late and i got a new clickity keyboard that i'm still adjusting to :)